packages/manager/apps/dedicated/client/app/dedicated/server/install/image/components/options/controller.js

```diff
@@ -1,5 +1,11 @@
 export default class DedicatedServerInstallImageOptionsCtrl {
   $onInit() {
-    [this.model.diskGroup] = this.server.hardware.specifications.diskGroups;
+    [this.model.diskGroup] =
+      !this.server.hardware.specifications.diskGroups ||
+      this.server.hardware.specifications.diskGroups.length <= 1
+        ? this.server.hardware.specifications.diskGroups
+        : this.server.hardware.specifications.diskGroups.sort((a, b) =>
+            a.description.localeCompare(b.description),
+          );
```

Comment:
**nitpick:** Code can be simplified
```suggestion
    [this.model.diskGroup] =
      !this.server.hardware.specifications.diskGroups &&
        this.server.hardware.specifications.diskGroups.sort((a, b) =>
            a.description.localeCompare(b.description),
          );
```

Reply:
I did an update. You were right, no need to check for the size of the array. But I didn't implement your solution because I tested it and in the case where input array is null, it was returning false which raises an error since it is not an iterable value.